docs/source/reference/exception.rst

```diff
@@ -85,11 +85,12 @@ The class hierarchy for TensorBay custom exceptions is::
          +-- FrameError
          +-- ResponseError
              +-- AccessDeniedError
+             +-- ForbiddenError
```

Comment:
the commit message is not clear,
you should describe the "change" with how or why.